### PR TITLE
Added validation to gynac-form

### LINKED ISSUE #280: Apply form-validation to Gynac help form
## Is your feature request related to a problem? Please describe.
YES, currently if no one enters a value then also it will submit the form instead of that it should throw an error

## Describe the solution you'd like
Make them compulsory attribute

### PR DESCRIPTION
**Pull request by garvitgupta13**
#### Description
Issue FIXES #280

#### Describe the changes you've made

Made the name and age field as compulsory field and on entering invalid age, contact-number, email, weight , etc will throw an error

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

image

#### Checklist

- [ ] I have added my name in the contributors list at the end of README.md file.
- [] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.
- [x] Open Source Program names(OPTIONAL: If you participated in any open-source program then please mention the program name here) `gssoc21`

### CODE REVIEW

File: periodo/components/Gynae/Gynac.js [hunk 1]
```diff
@@ -120,6 +124,7 @@ export default function Gynac() {
                   }}
                   variant="outlined"
                   label="Email"
+                  type="email"
                   name="email"
```
Comment (Aaishpra) [inline periodo/components/Gynae/Gynac.js:128]: Can we make this required too

File: periodo/components/Gynae/Gynac.js [hunk 2]
```diff
@@ -100,6 +102,8 @@ export default function Gynac() {
                   }}
                   variant="outlined"
                   label="Age"
+                  type="number"
+                  required
```
Comment (Aaishpra) [inline periodo/components/Gynae/Gynac.js:106]: Remove the required from here

File: periodo/components/Gynae/Gynac.js (hunk 1 above)
Comment (garvitgupta13) [reply periodo/components/Gynae/Gynac.js:127]: done

File: periodo/components/Gynae/Gynac.js (hunk 2 above)
Comment (garvitgupta13) [reply periodo/components/Gynae/Gynac.js]: done

**Comment by Aaishpra** [comment]
Great, looks good to me

**Pull request merged**